src/System.Text.Primitives/System/Text/Formatting/PrimitiveFormatter_time.cs

```diff
@@ -182,19 +182,19 @@ static bool TryFormatDateTimeFormatO(DateTimeOffset value, bool isDateTime, Span
         {
```

Comment:
We renamed the parameter formattingData to encodingData in the above TryFormatDateTimeFormat method, why not here?